src/test/kotlin/coverosR3z/timerecording/UIEditTimeTests.kt

```diff
@@ -80,7 +80,7 @@ class UIEditTimeTests {
     private fun enterInitialTime() {
         // when the employee enters their time
         enterTimeForEmployee("projectb")
-        pom.driver.get("${pom.domain}/${ViewTimeAPI.path}")
+        pom.driver.get("${pom.domain}/${ViewTimeAPI.path}?date=$DEFAULT_DATE_STRING")
```

Comment:
Some of our UI tests make changes on a different day than "today", so we have to jump to those days to do our work